web/src/services/s3/routes.ts

```diff
@@ -41,23 +41,6 @@ export default [
       }
     ]
   },
-  {
```

Comment:
Please check if it is there in swagger document also and remove it from there also.